tasks/ssl.yml

```diff
@@ -18,23 +18,41 @@
       mode: 0700
       recurse: yes
 
-  - name: Copy SSL certificate
-    copy:
-      src: "{{ site.server.ssl.src_dir }}/{{ site.server.ssl.cert }}"
-      dest: "{{ nginx_ssl_dir }}/{{ site.server.ssl.cert }}"
-      mode: 0600
-      remote_src: "{{ site.server.ssl.remote_src | default('no') }}"
-    notify:
-      - restart nginx
+  - block:
+    - name: Copy SSL certificate
+      copy:
+        src: "{{ site.server.ssl.src_dir }}/{{ site.server.ssl.cert }}"
+        dest: "{{ nginx_ssl_dir  }}/{{ site.server.ssl.cert }}"
+        mode: 0600
+        remote_src: "{{ site.server.ssl.remote_src | default('no') }}"
+      notify:
+        - restart nginx
 
-  - name: Copy SSL key
-    copy:
-      src: "{{ site.server.ssl.src_dir }}/{{ site.server.ssl.key }}"
-      dest: "{{ nginx_ssl_dir }}/{{ site.server.ssl.key }}"
-      mode: 0600
-      remote_src: "{{ site.server.ssl.remote_src | default('no') }}"
-    notify:
-      - restart nginx
+    - name: Copy SSL key
+      copy:
+        src: "{{ site.server.ssl.src_dir }}/{{ site.server.ssl.key }}"
+        dest: "{{ nginx_ssl_dir }}/{{ site.server.ssl.key }}"
+        mode: 0600
+        remote_src: "{{ site.server.ssl.remote_src | default('no') }}"
+      notify:
+        - restart nginx
+    when: site.server.ssl.create_symlink | default(False) == False
+  - block:
+    - name: Create SSL certificate symlink
+      file:
+        src: "{{ site.server.ssl.src_dir }}/{{ site.server.ssl.cert }}"
+        dest:  "{{ nginx_ssl_dir }}/{{ site.server.ssl.cert }}"
+        state: link
+      notify:
+        - restart nginx
+    - name: Create SSL key symlink
+      file:
+        src: "{{ site.server.ssl.src_dir }}/{{ site.server.ssl.key }}"
+        dest:  "{{ nginx_ssl_dir }}/{{ site.server.ssl.key }}"
+        state: link
+      notify:
+        - restart nginx
+    when: site.server.ssl.create_symlink | default(False) == True
```

Comment:
Same here. Maybe `site.server.ssl.create_symlink is defined and site.server.ssl.create_symlink == true`?